src/templates/post.module.sass

```diff
@@ -28,14 +31,20 @@
   align-items: baseline
   margin-bottom: 10px
 
+.post_headline
+  margin-bottom: 1.45em
+
 .post_ingredients
-  padding: 0 40px
+  padding: 0 20px
```

Comment:
I would go 25px with both ingredients and the preparation sections. Especially the prep one looks a bit too close to the edge.